src/planscape/conditions/load.py

```diff
@@ -30,10 +30,40 @@
     load.load_metrics('sierra_cascade_inyo')
 """
 
+def convert_metrics_nodata(region: str):
+    """Replaces the NoData pixels in metric rasters with the standard NaN value and updates the raster profile accordingly.
+
+    Args:
+      region: The region to rewrite
+    """
+    config_path = os.path.join(
+        PLANSCAPE_ROOT_DIRECTORY, 'src/planscape/config/conditions.json')
+    config = PillarConfig(config_path)
+
+    region = config.get_region(region)
+    for pillar in config.get_pillars(region):
+        if pillar['display']:
+            for element in config.get_elements(pillar):
+                for metric in config.get_metrics(element):
+                    # TODO: Update to interprted when available
+                    for metric_type in ['.tif', '_normalized.tif']:
+                        metric_is_raw = True if metric_type == '.tif' else False
+
+                        reader = ConditionReader()
+                        nan_condition = convert_metric_nodata_to_nan(
+                            reader, metric, ConditionScoreType.CURRENT, metric_is_raw)
+
+                        # Overwrite original local version
+                        base_filepath = os.path.join(os.path.dirname(
+                            PLANSCAPE_ROOT_DIRECTORY), metric['filepath']) + metric_type
+                        with rasterio.open(base_filepath,'w', nan_condition.profile) as dst:
+                            dst.write(nan_condition.raster, 1)
+
+
 def load_metrics(region: str):
     """Loads the metric rasters defined by the configuration into the database if display=true."""
     config_path = os.path.join(
-        PLANSCAPE_ROOT_DIRECTORY, 'Planscape/src/planscape/config/conditions.json')
+        PLANSCAPE_ROOT_DIRECTORY, 'src/planscape/config/conditions.json')
     config = PillarConfig(config_path)
 
     region = config.get_region(region)
```

Comment:
The depth of indentation is making it hard to follow (see other comments).  One suggestion: replay line 71 

```
  if pillar['display']:
    ...
```
with 
```
  if not pillar['display']:
    continue
```